app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ]  **New actions is missing** in your comments controller! Kindly ensure that the comments controller have  both _new_ and _create_ actions similar to the one shown below:


```ruby
# controllers/comments_controller.rb

class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end

	def create
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]

    if @comment.save
      redirect_to post_path(@comment.post)
    else
      @article = Post.find(params[:post_id])
      redirect_to post_path(@post)
    end
  end

  # remind students about the strong parameters in this lesson https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end

	...
end
```

Reply:
done, thanks!

Reply:
done, thanks!